scripts/direct_bookmark_save_test.py

```
@@ -0,0 +1,106 @@
+            혁신적인 발전을 이루고 있습니다. 데이터 사이언스와 빅데이터 
+            분석을 통해 패턴을 학습하고 예측 모델을 구축할 수 있습니다.
+            파이썬, TensorFlow, PyTorch 등의 도구들이 널리 사용됩니다.
+            """,
+            "keywords": ["인공지능", "머신러닝", "딥러닝", "데이터사이언스", "파이썬"],
+            "summary": "AI와 ML 기술의 전반적인 개요와 활용 분야 소개"
+        }
+        
+        # 정수형 user_id 사용 (문자열 변환 없이)
+        user_id = 456  # 직접 정수 사용
+        source_id = "direct_test_bookmark_003"
+        
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)